General Style Guide.md

```diff
@@ -35,6 +38,7 @@ The decision is not always straight forward and may require discussion with the
 | Prerogative(s) | Privilege(s) ||
 | Remembrance | Thankfulness | Only and only when specifically describing the Lord's Supper. |
 | Salvific | Saving _or_ Salvation ||
+| Theophany | Appearing ||
```

Comment:
I haven't found the context, but I think the "theo" bit needs to be in the translation "divine appearing"? "divine visitation" is pretty common.

Reply:
In the end, I did not change the corresponding sentence in the article and kept it as is because it was self-explanatory:
> The prophets describe the final establishment of God’s kingdom in terms of a theophany—a divine visitation

Therefore I suggest removing this line from the guidelines.

What do you think?

Reply:
Actually, no, there is a place where I changed it.

Original:
> God will finally break into history in a glorious theophany to establish his rule in all the earth.

Replacement:
> God will finally break into history in a glorious appearing to establish his rule in all the earth.

Reply:
Ok, that works for me